transitions/multi_form_alerts.js

```diff
@@ -125,33 +132,33 @@ const getPhonesWithDuplicates = (recipients, countedReports) => {
   return getPhonesOneRecipient(recipients, countedReports);
 };
 
-const validateConfig = (alert) => {
```

Comment:
It now validates all alerts, not just one alert, so the whole function body is now in a loop so it's 4 spaces instead of 2.